use-cases/meeting-scheduling/src/main/java/org/acme/meetingschedule/rest/DemoDataGenerator.java

```diff
@@ -0,0 +1,268 @@
+                new Meeting(String.valueOf(count++), "Fast track virtualization"),
+                new Meeting(String.valueOf(count++), "Cross sell multitasking"),
+                new Meeting(String.valueOf(count++), "Profitize one stop shop"),
+                new Meeting(String.valueOf(count++), "Transform braindumps"),
+                new Meeting(String.valueOf(count++), "Engage data mining"),
+                new Meeting(String.valueOf(count++), "Downsize policies"),
+                new Meeting(String.valueOf(count++), "Ramp up synergies"),
+                new Meeting(String.valueOf(count++), "On board user experience"),
+                new Meeting(String.valueOf(count++), "Reinvigorate B2B"),
+                new Meeting(String.valueOf(count++), "Strategize virtualization"),
+                new Meeting(String.valueOf(count++), "Fast track multitasking"),
+                new Meeting(String.valueOf(count++), "Cross sell one stop shop"),
+                new Meeting(String.valueOf(count++), "Profitize braindumps"),
+                new Meeting(String.valueOf(count++), "Transform data mining"),
+                new Meeting(String.valueOf(count++), "Engage policies"),
+                new Meeting(String.valueOf(count++), "Downsize synergies"),
+                new Meeting(String.valueOf(count++), "Ramp up user experience"),
+                new Meeting(String.valueOf(count++), "On board B2B"),
+                new Meeting(String.valueOf(count++), "Reinvigorate e-business"),
+                new Meeting(String.valueOf(count++), "Strategize multitasking"),
+                new Meeting(String.valueOf(count++), "Fast track one stop shop"),
+                new Meeting(String.valueOf(count++), "Cross sell braindumps"),
+                new Meeting(String.valueOf(count++), "Profitize data mining"),
+                new Meeting(String.valueOf(count++), "Transform policies"),
+                new Meeting(String.valueOf(count++), "Engage synergies"),
+                new Meeting(String.valueOf(count++), "Downsize user experience"),
+                new Meeting(String.valueOf(count++), "Ramp up B2B"),
+                new Meeting(String.valueOf(count++), "On board e-business"),
+                new Meeting(String.valueOf(count), "Reinvigorate multitasking"));
+        // Duration
+        List<Pair<Float, Integer>> durationGrainsCount = List.of(
+                new Pair<>(0.33f, 8),
+                new Pair<>(0.33f, 12),
+                new Pair<>(0.33f, 16));
+        durationGrainsCount.forEach(p -> applyRandomValue((int) (p.key() * meetings.size()), meetings,
+                m -> m.getDurationInGrains() == 0, m -> m.setDurationInGrains(p.value())));
+        // Ensure there are no empty duration
+        meetings.stream()
+                .filter(m -> m.getDurationInGrains() == 0)
+                .forEach(m -> m.setDurationInGrains(8));
+        // Attendants
+        MutableInt attendantCount = new MutableInt();
```

Comment:
If you don't use lambdas so much, you won't have to complicate your code with mutable references.